syntax = "proto3";

import "google/protobuf/timestamp.proto";

message Todo {
  int32 id = 1;
  string title = 2;
  string description = 3;
  bool completed = 4;
  google.protobuf.Timestamp created_at = 5;
  google.protobuf.Timestamp updated_at = 6;
  int32 user_id = 7;
}
